README.md

```diff
@@ -1 +1,38 @@
-# Hello-world
```

Comment:
Kindly use the README template for the [README file](https://github.com/microverseinc/readme-template) with its styling and add the same information you added to that template.